setup.cfg

```diff
@@ -54,6 +55,7 @@ dev =
     ipdb==0.13.3
     ipython==7.16.1
     isort==4.3.21
+    msgpack===1.0.0
```

Comment:
do we want to add this dependency or can we impose json?

Reply:
This is just a dev dependency to be able to run the tests.

Let our users choose their favorite serialization library :-)

Reply:
I didn't realize it was just a dev dependency. Good with me.